### DISCUSSION TITLE
Shouldn't a failed test also be shown as zero speed?

### DISCUSSION THREAD

**Discussion by tomhusband** [runtipi/runtipi-appstore#5081] [Q&A] [answered]
The other day our internet went down for a several hours. MySpeed showed the failed tests but not anything about no internet. I expected to see zero speed for every attempted test on the graph during the outage.

Maybe I'm missing a setting?

**Comment by steveiliop56** [comment, marked as answer]
Hello,
Since we are not the developers of the app we can't really help you. Please move the discussion on their Github.
